client/my-sites/checkout/checkout/redirect-payment-box.jsx

```diff
@@ -215,6 +239,35 @@ export class RedirectPaymentBox extends PureComponent {
 		} );
 	}
 
+	renderWechatScanText() {
+		if ( cartValues.cartItems.hasRenewalItem( this.props.cart ) ) {
+			return translate( 'Scan the barcode to confirm your %(price)s subscription with %(paymentProvider)s.', {
+				args: {
+					price: this.props.cart.total_cost_display,
+					paymentProvider: this.getPaymentProviderName(),
+				},
+				context: 'Instruction to scan a QR barcode and finalize payment with wechat for a subscription service.',
+			} );
+		}
+
+		return translate( 'Scan the barcode to confirm your %(price)s purchase with %(paymentProvider)s.', {
+			args: {
+				price: this.props.cart.total_cost_display,
+				paymentProvider: this.getPaymentProviderName(),
+			},
+			context: 'Instruction to scan a QR barcode and finalize payment with WeChat Pay.',
+		} );
+	}
+
+	renderWechatRedirectText() {
+		return translate( 'On mobile? To open and pay with the WeChat Pay app directly {{a}}click here{{/a}}.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?